#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "steering"

gen = ParameterGenerator()

gen.add(
    "min_possible_steering_angle",
    double_t,
    0,
    "Specifies the min possible steering value (0, 0.99) that the car should be able to steer at every speed.",
    0.3,
    0.0001,
    0.99
)

gen.add(
    "max_steering_angle",
    double_t,
    0,
    "Max steering angle of the car.",
    30,
    0.0001,
    50
)

exit(gen.generate(PACKAGE, "steering", "steering"))